portality/static/widget/fixed_query_src_edges.js

```diff
@@ -0,0 +1,68 @@
+jQuery(document).ready(function($) {
+
+    $.extend(true, doaj, {
+
+    publicSearch : {
+        activeEdges : {},
+
+        buildQuery: function(params){
+            let query;
+            if (params.rawQuery){
+                query = es.newQuery({raw: params.queryString});
+            }
+            else {
+                query = es.newQuery(params.initialParams);
+                query.setQueryString(es.newQueryString(params.queryString));
+                if(params.hasOwnProperty(`sortFilter`)) {
+                    query.setSortBy(es.newSort(params.sortFilter));
+                }
+                if(params.hasOwnProperty(`es_type`)) {
+                    query.addMust(es.newTermFilter({field: "es_type", value: params.es_type}));
+                }
+            }
+            return query;
+        },
+
+        init : function(params) {
+            if (!params) { params = {} }
+
+            let current_domain = document.location.host;
+            let current_scheme = window.location.protocol;
```

Comment:
I don't think this is going to work, is it?  Won't this create a URL that's localised to the user's URL rather than the doaj url?